website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -108,20 +109,55 @@ To delete a published tag to re-tag, follow these steps:
     git push -d origin vx.y.z
 ```
 
-## Deploying ORCA buckets, RDS cluster and Cumulus ORCA modules in bamboo
+## Deploying ORCA buckets, RDS cluster and Cumulus ORCA modules via bamboo
 
-For testing purposes, you should use your feature branch in cumulus-orca github repo and `prototype-latest` bamboo plan so that it does not affect the ORCA github `develop` branch. The `prototype-latest` bamboo plan is linked to `feature/ORCA-test-bamboo` github branch as the default branch under cumulus-orca repo. In addition, if changes are made to the bamboo spec file `bamboo.yaml` in this default branch, you have to manually import the bamboo spec by choosing `ORCA-test-branch` as the linked repo.
+For testing purposes, you should use your feature branch in cumulus-orca github repo and [`prototype-latest`](https://ci.earthdata.nasa.gov/browse/ORCA-PL) bamboo plan so that it does not affect the ORCA github `develop` branch. The `prototype-latest` bamboo plan is linked to `feature/ORCA-test-bamboo` github branch as the default branch under cumulus-orca repo. In addition, if changes are made to the bamboo spec file `bamboo.yaml` in this default branch, you have to manually import the bamboo spec by choosing `orca test branch` as the linked repo.
 
-Make sure to use the `ORCA Deploy Plan` bamboo plan for deploying the resources. These are the ORCA buckets that are created in disaster recovery AWS account:
+:::warning
+You should reset `feature/ORCA-test-bamboo` before using it.
+1. Rename `feature/ORCA-test-bamboo` to `feature/ORCA-test-bamboo-old`
+1. Create a new branch based off of your branch named `feature/ORCA-test-bamboo`
+1. In the new branch's `bamboo.yaml`:
+   Delete all but one of the plans.
+   Change plan's `name` to `prototype-latest`.
+   In each `repositories` element, change `orca-develop` to `orca test branch`.
+   In each `plan` element, change `OI`/`ODP` in `key` values to `PL`
+:::
+
+:::warning
+`prototype-latest` exposes an ordering issue where the release stage must be run before deployment/integration checks can be run.
+DO NOT RUN THE RELEASE STAGE FROM `PROTOTYPE-LATEST`
+Comment the release stage out in `bamboo.yaml` at the top of the file, and under `stages:`. Note that indentation is not a reliable indicator of block length, so make sure that all release code, including `repositories`, `triggers`, and `branches`, are commented out.
+:::
+

```

Comment:
Better to just remove the release stage from the `prototype-latest` plan and not needing to update the doc here

Reply:
Unless it is regularly synced with `develop` (not via manual, prone-to-error updates) it is invalid as a testing environment.